### DISCUSSION TITLE
Getting started with Tycho?

### DISCUSSION THREAD

**Discussion by rfolo9li** [eclipse-tycho/tycho#979] [Q&A]
Hi,

I have to convert existing Eclipse plugins to Maven projects using Tycho. But I'm completely new to Tycho and don't know where to start. What do I have to do? Isn't there an official "getting started guide" or the like?

Best regards,
Ed

**Comment by laeubi** [comment]
You could start to setup your build with Tycho pomless, this requires just a few steps to build your eclipse projects with maven:

http://blog.vogella.com/2019/11/25/pom-less-tycho-enhanced/

**Reply by rfolo9li** [reply]
Thanks for the information, the POM-less feature looks really promising. However, I still don't know how to start.

I need some documentation on how to use Tycho and getting started from **zero knowledge**. I'm asking myself the following questions right now:
1. How does a minimal setup for an Eclipse plugin (hello world example) look like?
2. What Tycho modules/plugins do I need in which use case?
3. What are the best practices (project structure, configuration or the like)?

**Reply by laeubi** [reply]
You can take a look at the demo projects:

- https://github.com/eclipse/tycho/tree/master/demo
- https://github.com/vogellacompany/tycho-example
